### PR TITLE
fs-1277 Removed bug in jinja

### PR DESCRIPTION
**Pull request by donshellywork**
this leads from fs-1195 where a bug was introduced

https://digital.dclg.gov.uk/jira/browse/FS-1277

### CODE REVIEW

File: .github/workflows/govcloud.yml [hunk 1]
```diff
@@ -90,7 +90,7 @@ jobs:
           CF_DOCKER_PASSWORD=${{ secrets.GHCR_PAT }} cf push funding-service-design-form-runner-dev -f ./manifest.yml \
             --docker-image ${{ env.DOCKER_IMAGE_TAG }} \
             --docker-username ${{ github.actor }}
-
```
Comment (gidsg) [inline .github/workflows/govcloud.yml:93]: can we revert these whitespace only changes?

File: manifest.yml [hunk 2]
```diff
@@ -9,7 +9,7 @@ applications:
     - route: forms.dev.fundingservice.co.uk
   env:
     PREVIEW_MODE: TRUE
-
```
Comment (gidsg) [inline manifest.yml:12]: can you revert these whitespace only changes?

File: .github/workflows/govcloud.yml (hunk 1 above)
Comment (gidsg) [reply .github/workflows/govcloud.yml:93]: Please revert to the version of this file from the main branch e.g. `git checkout main -- .github/workflows/govcloud.yml`

**Review by gidsg — APPROVED** [review]
LGTM 👍

**Review by SentientHousePlant — APPROVED** [review]

**Pull request merged**